src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+standard. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images run the in the cluster are
+signed and verified, proving that they come from their stated authors, and
+haven't been tampered with (for other concerns, see the docs on [how we implement a Secure Supply Chain for the policies themselves](TODO link to https://github.com/kubewarden/docs/pull/100/files)).
+
+Sigstore signatures of containers are stored as layers of the container image.
+Policies that want to check Sigstore signatures of containers need then to check
+those container layers, and would need to pull the signature layers to see the
+signatures themselves.
+
+Obtaining and operating with those container layers needs to happen outside of
+the waPC guest (the policy), and on the waPC host (Kubewarden's
+`policy-server`).
```

Comment:
I usually prefer to avoid the usage of waPC:

```suggestion
Obtaining and operating with those OCI layers needs to happen outside of
the WebAssembly guest (the policy). Hence this is done by the WebAssembly runtime: Kubewarden's
`policy-server` or `kwctl.
```

I also prefer to avoid the usage of "container" because this can work with any kind of OCI layer. Our primitives can work for verification of container images, kubewarden policies, helm charts and generally speaking any kind of OCI artifact.